sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        /// <summary>
+        /// This sample illustrates how to consume Bounding Box type data that is available across the Form Recognizer library.
```

Comment:
Is it worth including in the introductory comment a description of what a Bounding Box is?  Some people don't know this when they first come to the library.  Also worth noting units and where they're found.

Reply:
hmm great point. Will add